dydx/dydxPresenters/dydxPresenters/_v4/Onboarding/Connect/dydxOnboardConnectViewBuilder.swift

```diff
@@ -56,9 +59,12 @@ private class dydxOnboardConnectViewPresenter: HostedViewPresenter<dydxOnboardCo
         return viewModel
     }()
 
-    private let walletSetup = dydxV4WalletSetup()
+    private let walletSetup: dydxV4WalletSetup
+
+    init(onboardingAnalytics: OnboardingAnalytics, walletSetup: dydxV4WalletSetup) {
```

Comment:
```suggestion
    init(onboardingAnalytics: OnboardingAnalytics = OnboardingAnalytics(), walletSetup: dydxV4WalletSetup = OnboardingAnalytics()) {
```

to reduce boiler plate init when not testing